src/main/resources/assets/fabrishot/lang/fi_fi.json

```diff
@@ -0,0 +1,12 @@
+{
+  "fabrishot.config.title": "Fabrishot",
+  "fabrishot.config.category": "Kuvankaappauksen mitat",
+  "fabrishot.config.override_screenshot_key": "Korvaa tavallinen kuvankaappaus Fabrishotilla",
+  "fabrishot.config.custom_filename_format": "Käytä eri tiedostonimen formaattia",
```

Comment:
Could you also translate the new `fabrishot.config.custom_file_name` and `fabrishot.config.custom_file_name.tooltip` entries?

Reply:
Sure, will push in a bit

Reply:
Ah never mind, someone was faster